src/base/reset.scss

```diff
@@ -20,3 +20,23 @@ img {
   height:    auto;
   border:    none;
 }
+
+::-webkit-input-placeholder {
+  color:     scaleColor($ts-black, 6);
+  font-size: $tu-small-fontSize;
```

Comment:
What is the default font-size? Does it not inherit?

Reply:
oh that was just copied from classic - I'm cool with letting it inherit